### PR TITLE
GOOD-98: Update deps

### PR DESCRIPTION
**Pull request by ghsteff**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v18.12.0
```
Comment (ghsteff) [inline .nvmrc:1]: Added this for `vite` and `lint-staged` which now require node 18+

**Comment by amcdnl** [comment]
Can you fix the build too?

**Comment by amcdnl** [comment]
No yarn here - lets use npm

**Comment by ghsteff** [comment]
> No yarn here - lets use npm

Sounds good

**Review by amcdnl — APPROVED** [review]

**Pull request merged**